src/ILCompiler.TypeSystem/src/TypeSystem/Interop/IL/NativeStructType.Mangling.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Internal.TypeSystem.Interop
+{
+    public partial class NativeStructType : MetadataType, IPrefixMangledType
```

Comment:
Nit: I've been avoiding repeating the base type and visibility on the dot-files because it causes unnecessary frustrations when some of that changes, but I'm not sure how much of a convention we want that to be.